src/index.ts

```diff
@@ -96,17 +116,6 @@ const transformer = (_: ts.Program) => (context: ts.TransformationContext) => (
     node.arguments.length === 1;
 
   function visit(node: ts.Node): ts.VisitResult<ts.Node> {
```

Comment:
## Note on TS changes regarding type only exports

It seems that types only export behaviour has changed in TypeScript. For one, they added `isTypeOnly`, but they've also changed the standard behaviour of `export * from 'module'` and `import 'module'`

Even if `module` has only type exports, this line will still get compiled into the final result. This was causing the tests to fail, by default, after cloning the package. In order to match the prescribed TS behaviour , we've added the `isTypeOnly` param where needed as well as removed the following conditional.

In short, this means the TS compiler has taken over this logic.